raster/r.info/main.c

```diff
@@ -436,7 +436,7 @@ int main(int argc, char **argv)
 		mean = (double)(rstats.sum / rstats.count);
 		sd = sqrt(rstats.sumsq / rstats.count - (mean * mean));
 
-		fprintf(out, "n=%jd\n", rstats.count);
+		fprintf(out, "n=%jd\n", (intmax_t)rstats.count);
```

Comment:
As above, explicit cast should not be needed.

Reply:
Changed to grass_int64. Changed format specifier, now using PRId64.